### PR TITLE
Feat: MAIN PAGE (RSS-ECOMM-2_17)

### PR DESCRIPTION
**Pull request by JeckJonnyQ**
### What type of PR is this?

- [x] 🌟 Feature
- [ ] 🛠 Bug Fix
- [ ] ⚙️ Refactor
- [ ] 📖 Documentation

### Description

- **Brief Overview:**
  
- **Implementation Approach:**
  

### Additional Information

- **Screenshots/Links:**
  
- [ ] Related Issues:

### Checklist

- [ ] ✅ I have performed a self-review of my own code.
- [ ] 📝 I have commented my code, particularly in hard-to-understand areas.
- [ ] 🔧 I have made corresponding changes to the documentation (if applicable).
- [ ] 🚫 My changes generate no new warnings or errors.

### CODE REVIEW

File: src/pages/MainPage/MainPage.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import "./MainPage.scss";
+import React, { useState, ReactElement } from "react";
```
Comment (Oleg-Melnikow) [inline src/pages/MainPage/MainPage.tsx]: Please remove import `useState` if you don't use it.

**Review by Oleg-Melnikow — COMMENTED** [review]
Please check it

File: src/pages/MainPage/MainPage.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+          <div className="header__nav">
+            <NavLink className="header__nav_link-home" to="/">
+              Home
+            </NavLink>
+            <NavLink className="header__nav_link-catalog" to="/catalog">
+              Catalog
+            </NavLink>
+            <NavLink className="header__nav_link-about" to="/about">
+              About
+            </NavLink>
```
Comment (Oleg-Melnikow) [inline src/pages/MainPage/MainPage.tsx]: can we use in this place `map`  to get rid of duplication, for example:
`['home', 'catalog', 'about'].map((el) => {
  return <NavLink> {el} </NavLink>
} )`

and create variable for this array

File: src/pages/MainPage/MainPage.scss [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        width: 100%;
+      }
+
+      .header__nav {
+        @include DisplayFlex;
+        flex-grow: 0.25;
+
+        .header__nav_link-home,
+        .header__nav_link-catalog,
+        .header__nav_link-about {
```
Comment (Oleg-Melnikow) [inline src/pages/MainPage/MainPage.scss:34]: can we use general class instead of these classes?

**Review by Oleg-Melnikow — COMMENTED** [review]
Nice job! Look at the comments

File: src/pages/MainPage/MainPage.tsx (hunk 1 above)
Comment (JeckJonnyQ) [reply src/pages/MainPage/MainPage.tsx]: Completed.

File: src/pages/MainPage/MainPage.tsx (hunk 2 above)
Comment (JeckJonnyQ) [reply src/pages/MainPage/MainPage.tsx]: Completed

**Review by AndreiMakhnach2209 — APPROVED** [review]

**Review by Oleg-Melnikow — APPROVED** [review]
Good job!

**Pull request merged**